css/style.css

```diff
@@ -2,7 +2,7 @@
 @import url('/Page/Login/login.css');
 @import url('/Page/Logout/logout.css');
 @import url('/Page/Signup/signup.css');
-@import url("https://robust-ui.netlify.app/css/styles.css")
+@import url("https://robust-ui.netlify.app/css/styles.css");
 @import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;0,900;1,200&display=swap");
```

Comment:
just a suggestion that you can import the font in either typography.css or utility.css

Reply:
ok